app/controllers/authorization_requests_controller.rb

```diff
@@ -107,4 +111,8 @@ def authorization_request_class
   def extract_authorization_request_form
     @authorization_request_form = AuthorizationRequestForm.find(params[:form_uid])
   end
+
+  def authorization_exists?
+    current_user.current_organization.authorization_requests.where(type: authorization_request_class.to_s).any?
```

Comment:
Y'a un `current_organization` comme méthode

```suggestion
    current_organization.authorization_requests.where(type: authorization_request_class.to_s).any?
```